SignalServiceKit/src/Storage/TSYapDatabaseObject.m

```diff
@@ -223,6 +223,17 @@ - (void)applyChangeToSelfAndLatestCopy:(YapDatabaseReadWriteTransaction *)transa
     }
 }
 
+- (void)reload
+{
+    TSYapDatabaseObject *latest = [[self class] fetchObjectWithUniqueID:self.uniqueId];
+    if (!latest) {
+        OWSFail(@"%@ in %s `latest` was unexpectedly nil", self.logTag, __PRETTY_FUNCTION__);
+        return;
+    }
+
+    [self setValuesForKeysWithDictionary:latest.dictionaryValue];
```

Comment:
Huh.  Since we often put logic (e.g. migration logic) in `initWithCoder:`, I wonder if reloading this way will be always be safe.